common/scala/src/main/resources/application.conf

```diff
@@ -267,6 +267,18 @@ whisk {
         }
     }
 
+    yarn {
```

Comment:
Brief documentation about the various config option would be useful

Reply:
Did you see the markdown page that I added? https://github.com/apache/incubator-openwhisk/blob/23f47ae4ac82c65eac62e20436c4dd2dd2121881/docs/yarn.md

Reply:
So far the convention was to add a brief inline description in conf file itself. Can you add a comment here referring to final url of yarn.md .

Reply:
Sorry. I misunderstood. I added comments to the application.conf file